src/components/activity/ActivityRegistrator.tsx

```diff
@@ -10,36 +10,101 @@ import { ChoiceBasedTextInput } from '../ChoiceBasedTextInput';
 
 import { useTranslation } from 'language/LanguageProvider';
 import Storage from 'storage';
-import { ActivitiesEntry } from 'storage/types';
+import { ActivitiesDay, ActivitiesEntry } from 'storage/types';
 
 import { ISODate } from 'utils';
+import { getDaysInMonth } from 'react-native-paper-dates/lib/typescript/src/Date/dateUtils';
 
 export const ActivityRegistrator = ({ route, navigation }: any) => {
```

Comment:
I do feel like we should add typing to these components...

Reply:
I agree, might do add it to this PR if i have the time. Otherwise i'll add an issue for it.